lib/elixir/test/elixir/macro_test.exs

```diff
@@ -453,6 +453,30 @@ defmodule MacroTest do
              """
     end
 
+    test "with block of code" do
```

Comment:
Perhaps we can add a test for the empty block of code (although it isn't really useful, just make sure it doesn't crash)

Reply:
Or in the pattern we could exclude it by matching on non-empty `exprs`?

<img width="334" alt="Screenshot 2024-05-30 at 7 01 37" src="https://github.com/elixir-lang/elixir/assets/11598866/e4a5c323-113e-4056-97ca-9df7e20ed238">

Reply:
I think matching on empty is good idea. Formatter is fixing this and compiler is throwing a warning anyway.